src/data.js

```diff
@@ -97,25 +97,13 @@ export function findAll (cozy, doctype) {
 
     return cozyFetchJSON(cozy, 'POST', path, {})
     .then((resp) => {
-      const result = {}
-      result.docs = []
+      const docs = []
 
       for (const row of resp.rows) {
         const { doc } = row
-        result.docs.push(doc)
+        docs.push(doc)
       }
-      return result
-    })
-    .catch((error) => {
```

Comment:
Why is the `catch` removed ?

Reply:
Because, like a `.query()` using an index, we don't have to catch the error, it should be caught by the client directly. It was caught in `findMany` because we include errors in doc in the result object. Here we directly return the array.